src/usr/local/www/system_advanced_network.php

```diff
@@ -84,6 +85,13 @@
 			unset($config['system']['prefer_ipv4']);
 		}
 
+		$_POST['global-v6duid'] = strtolower(str_replace("-", ":", $_POST['global-v6duid']));
+		if (($_POST['global-v6duid'] && !is_duid($_POST['global-v6duid']))) {
```

Comment:
If you check before call strtolower, this check can be removed and only is_duid() validation remain